contract/test/Bank.spec.ts

```diff
@@ -26,88 +26,155 @@ describe('Bank', () => {
     let member1: SignerWithAddress
     let member2: SignerWithAddress
 
-    context('I want to deploy my Bank Smart Contract', () => {
-      before(async () => {
-        ;[owner, contractor, member1, member2] = await ethers.getSigners()
-        await deployContract(owner)
-      })
+    before(async () => {
+      ;[owner, contractor, member1, member2] = await ethers.getSigners()
+      await deployContract(owner)
+    })
```

Comment:
Extract getting signers and deploying contract and make it available for every context